src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/layout.scss

```diff
@@ -1,3 +1,7 @@
+// Colors
+$shrinkBackgroundColor: #21272E !default;
```

Comment:
Every color is configured in husky. Do you want me to add a color variable in husky or just add one at the top of this file?